usertools/dpdk-telemetry-watcher.py

```
@@ -0,0 +1,435 @@
+
+        # Map common shortcuts to actual field names
+        field_map = {
+            "rx": "ipackets",
+            "tx": "opackets",
+        }
+        field = field_map.get(field, field)
+
+        # Get list of ethernet devices
+        process, port_list = query_telemetry(process, "/ethdev/list")
+        if not isinstance(port_list, list):
+            print(f"Error: Failed to get ethernet device list", file=sys.stderr)
```

Ruff (0.14.10)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)